pyopensn/src/pyapi.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Created on Sat, February 22
+ *
+ * Copyright (c) 2025 The OpenSn Authors <https://open-sn.github.io/opensn/>
+ */
+#ifndef PYOPENSN_PYAPI_HPP_
+#define PYOPENSN_PYAPI_HPP_
+
+#include <cstddef>  // std::size_t
+#include <vector>   // std::vector
+
+#include <mpi.h>
+
+#include <pybind11/pybind11.h>
+namespace py = pybind11;
+
+#include "framework/parameters/parameter_block.h"   // opensn::ParameterBlock
+#include "framework/parameters/input_parameters.h"  // opensn::InputParameters
+
+namespace opensn {
+
+// Environment for running Python
+// ------------------------------
+
+/** @brief Environment initializer and finalizer for OpenSn.*/
```

Comment:
The coding standards says:
- single line doco-comment use `///`
- multi-line use `/** */`

[See](https://open-sn.github.io/opensn/devguide/coding_standard.html#comments)